src/app/dashboard/whitelist/WhiteListPage.js

```diff
@@ -0,0 +1,378 @@
+//@flow
+
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import DocumentTitle from 'react-document-title'
+import uuidv4 from 'uuid/v4'
+import {
+  DataTable,
+  PaginationV2,
+  Modal,
+  ModalWrapper,
+  DatePicker,
+  DatePickerInput,
+  FileUploaderButton,
+} from "carbon-components-react"
+
+import type { Investor } from 'polymathjs/types'
```

Comment:
Please import external types after external imports and internal types after internal imports. Probably it's possible to add such rule in our eslint-plugin too.

Reply:
could not find an eslint rule for this specifically. i will do it manually

Reply:
manually fixed